### DISCUSSION TITLE
Select matching strings keyboard shortcut?

### DISCUSSION THREAD

**Discussion by Fatbat** [ajaxorg/ace#4628] [Q&A] [answered]
Normally, in Sublime or VS Code or whatever, selecting matching strings to the one currently selected is done by pressing CTRL D repeatedly until they are all selected, but in this editor that erases the entire line your selection is in.

Is there a keyboard shortcut to select matching items to the one you have selected?

**Comment by nightwing** [comment, marked as answer]
Yes ctr-alt-right/left arrows do that. 
With shift they skip the current selection like ctrl-k ctrl-d in vscode.

Also there are sublime and vscode keybindings if you want to use ctrl-d.

**Reply by Fatbat** [reply]
Thank you so much @nightwing!
